Extension/src/LanguageServer/extension.ts

```diff
@@ -322,6 +322,13 @@ function onAddToIncludePath(path: string): void {
     }
 }
 
+function onToggleSnippets(): void {
+    onActivationEvent();
+    // This only applies to the active client.
+    let settings: CppSettings = new CppSettings(clients.ActiveClient.RootUri);
+    settings.toggleSetting("snippets", "Enabled", "Disabled");
```

Comment:
I see that snippets are enabled via package.json.  How do you actually turn them off?

Reply:
The toggle does not seem to turn them off.

From my google-fu, I don't think theres a good way from VSCode to turn them off besides reading the package.json and deleting the snippet entry from it and reload VSCode.

Reply:
So when the settings changes we can re-write the package.json by adding/removing the snippet and optionally show a reload prompt? Otherwise, we might get complaints from users who want to use the other snippets extension instead (or just use no snippets).

Reply:
Yep. We get to bring this back :)

```
export function showReloadPrompt(): void { 
   let reload: string = "Reload"; 
   vscode.window.showInformationMessage("Reload the window to finish installing the C/C++ extension.", reload).then(value => { 
      if (value === reload) { 
         vscode.commands.executeCommand("workbench.action.reloadWindow"); 
      } 
   }); 
} 
```